docs/sources/installation/upgrading.md

```diff
@@ -58,29 +58,44 @@ restore:
 > psql grafana < grafana_backup
 ```
 
-### Ubuntu / Debian
+### Ubuntu or Debian
 
-If you installed grafana by downloading a debian package (`.deb`) you can just follow the same installation guide
-and execute the same `dpkg -i` command but with the new package. It will upgrade your Grafana install.
+You upgrade Grafana by following the same procedure as when you installed it.
 
-If you used our APT repository:
+#### Upgrade Debian package
+
+If you installed Grafana by downloading a Debian package (`.deb`), then you can execute the same `dpkg -i` command but with the new package. It will upgrade your Grafana installation.
+
+Go to the [download page](https://grafana.com/grafana/download?platform=linux) for the latest download
+links.
+
+```bash
+wget <debian package url>
+sudo apt-get install -y adduser libfontconfig1
+sudo dpkg -i grafana_<version>_amd64.deb
+```
+
+#### Upgrade from APT repository
+
+If you installed Grafana from our APT repository:
 
 ```bash
 sudo apt-get update
 sudo apt-get install grafana
 ```
 
-#### Upgrading from binary tar file
+#### Upgrade from binary .tar file
 
-If you downloaded the binary tar package you can just download and extract a new package
-and overwrite all your existing files. But this might overwrite your config changes. We
-recommend you place your config changes in a file named `<grafana_install_dir>/conf/custom.ini`
-as this will make upgrades easier without risking losing your config changes.
+If you downloaded the binary .tar package, then you can just download and extract a new package
+and overwrite all your existing files. However, this might overwrite your config changes. 
+
+We recommend that you save your custom config changes in a file named `<grafana_install_dir>/conf/custom.ini`.
+This allows you to upgrade Grafana without risking losing your configuration changes.
```

Comment:
Do we have this same recommendation in the install guide?

Reply:
could be good to have

Reply:
If they are installing for the first time, then they won't have config changes to save. This is only relevant when upgrading Grafana.